indico_toolkit/snapshots/snapshot.py

```diff
@@ -189,6 +189,41 @@ def get_all_labeled_text(
                 all_labeled_text.extend(text_found)
         return all_labeled_text
 
+    def get_label_count(self, by_document: bool = True):
+        """
+        Get the quantity of occurances each label in the label-set. 
+        Args:
+            by_document (bool, optional): if True, returns quantities of documents with a given label.
+                                                if False, returns quantities of total number of labels within the label-set.
+                                                Defaults to True.
+        """
+        label_count = dict()
+        for label in self.get_extraction_label_names():
+            label_count[label] = len([i for i in self.get_all_labeled_text(label, by_document) if i]) # if i only important for by_doc == True
+        return label_count
+
+    def get_file_names(self, label_name: str, with_label: bool = True):
+        """
+        Get the file names of all files which are either tagged or not tagged with a given label. 
+        Args:
+            label_name (str): name of the label
+            with_label (bool, optional): if True, returns a set of file names of all files tagged with label.
```

Comment:
we may want to name this variable something like `contains_label` to be more clear.